lego/apps/articles/fixtures/development_articles.yaml

```diff
@@ -74,3 +78,65 @@
       - lorem
       - ipsum
     youtube_url: https://www.youtube.com/watch?v=bLHL75H_VEM
+
+- model: articles.Article
+  pk: 4
+  fields:
+    title: Weekly - UKE 37
+    cover: test_article_cover.png
+    description: Takket være de flinke medlemmene i itDAGENE så fikk vi en pangstart
+      på uken med tre-retters middag.
+    text: Takket være de flinke medlemmene i itDAGENE så fikk vi en pangstart på
```

Comment:
Should this be html?

Reply:
The old fixtures are saved as normal text but I see that events save their `text` as html so we can do it here as well. Both get text from from the `Content` model, so it sould work just fine